package.json

```diff
@@ -28,14 +30,15 @@
     "react-router-dom": "~6.3.0",
     "react-scripts": "~5.0.1",
     "styled-components": "~5.3.5",
+    "ts-jest": "^28.0.7",
     "ts-node": "~10.8.1",
     "typescript": "~4.7.3",
     "web-vitals": "~2.1.4"
   },
   "scripts": {
     "start": "ts-node src/data/process_data.ts && react-scripts start",
     "build": "ts-node src/data/process_data.ts && react-scripts build",
-    "test": "ts-node src/data/process_data.ts && react-scripts test",
+    "test": "react-scripts test",
```

Comment:
undo this change now that the data is used in the tests explicitly, we want to make sure it's always created & available & in the correct format!

Reply:
done 👍